tests/utilities/test_dataframe_functions.py

```diff
@@ -230,6 +230,22 @@ def test_positive_rolling(self):
         self.assertListEqual(list(df["b"].values),
                              [5, 5, 6, 5, 6, 7, 12, 5, 6, 7, 8, 12, 13])
 
+        df = dataframe_functions.roll_time_series(df_full, column_id="id", column_sort="time",
```

Comment:
i think we should add more test (e.g. in the other direction and with maximum_number_of_timeshifts>1)